components/wmtk_components/embedded_component/internal/EmbeddedRemeshing.cpp

```diff
@@ -0,0 +1,117 @@
+#include "EmbeddedRemeshing.hpp"
+
+namespace wmtk::components::internal {
+EmbeddedRemeshing2D::EmbeddedRemeshing2D(
+    Eigen::MatrixXi& E_,
+    Eigen::MatrixXd& V_,
+    double blank_rate_,
+    double resolute_area_)
+    : E(E_)
+    , V(V_)
+    , blank_rate(blank_rate_)
+    , resolute_area(resolute_area_)
+{
+    F.resize(0, 0);
```

Comment:
You are in the constructor here. `F` was not used before so it already should have size `(0,0)` right?